data/strings/types_strings.txt

```diff
@@ -18189,21 +18190,106 @@
     ko = 베어 락
     mr = खडक
     nb = Bar stein
-    nl = Kale steen
+    nl = Kale rots
     pl = Odkryte skały
     pt = Zona rochosa
     ro = Piatră goală
-    ru = Скалы
+    ru = Каменная порода
     sk = Holá skala
     sv = Kala stenar
     sw = Mwamba tupu
     th = เปลือยร็อค
     tr = Çıplak Kaya
-    uk = Скелi
+    uk = Кам'яна порода
     vi = Đá trần
     zh-Hans = 岩石
     zh-Hant = 岩石
 
+  [type.natural.shingle]
+    comment = A land cover consisting of rounded rock fragments (pebbles) on a beach or riverbed.
+    en = Shingle
+    af = Gordelroos
+    ar = تقرير
+    az = Şingle
+    be = Галька
+    bg = Камъчета
+    ca = Còdol
+    cs = Oblázky
+    da = Småsten
+    de = Kieselsteine
+    el = Βότσαλα
+    es = Guijarros
+    es-MX = Guijarros
+    et = Pebbles
+    eu = Harri koskor
+    fa = گزارش
+    fi = Pebbles
+    fr = Galets
+    he = אבנים
+    hi = प्रतिवेदन
+    hu = Kavicsok
+    id = Kerikil
+    it = Ciottoli
+    ja = 小石
+    ko = 페블
+    lt = Akmenukai
+    mr = अहवाल
+    nb = Småstein
+    nl = Kiezels
+    pl = Kamyki
+    pt = Seixos
+    ro = Pietriș
+    ru = Галька
+    sk = Kamienky
+    sv = Småsten
+    th = รายงาน
+    tr = Çakıl Taşları
+    uk = Галька
+    vi = ván lợp
```

Comment:
Vietnamese has capital letters, why here and below the first letter is a small one?

Reply:
that's how the translation script returns it (from Google) and I missed it..

Reply:
fixed, thanks!